src/Native/Unix/System.Native/pal_console.c

```diff
@@ -167,63 +167,62 @@ static int TranslatePalControlCharacterName(int name)
     switch (name)
     {
 #ifdef VINTR
-        case PAL_VINTR: return VINTR;
+    case PAL_VINTR: return VINTR;
```

Comment:
Why are we changing this formatting?  The general style we've used is for cases to be indented.  If we're going to enforce a style, it should be that.

Reply:
I've asked for that - the new way is what is used all over coreclr repo and what was used in many of the places in corefx repo too.

Reply:
😦